doc/projektivelho.md

```diff
@@ -0,0 +1,56 @@
+# Projektivelho
```

Comment:
Projektivelhon dokumentaatiosta on oma tikettinsä mutta raapaisin tänne tuon luokkakaavion jo, koska poistin skema-kuvat database-dokkarista (ne on työläitä ylläpitää ja vähän turhia).